src/System/Console/GetOpt/Generics.hs

```diff
@@ -57,10 +58,15 @@ import           System.Console.GetOpt.Generics.Internal
 --     with exit-code @0@.) Help output is written to @stdout@.
 getArguments :: forall a . (Generic a, HasDatatypeInfo a, All2 Option (Code a)) =>
   IO a
-getArguments = do
+getArguments = getArgumentsMods []
+
+-- | Like 'getArguments` but allows to pass in 'Modifier's.
```

Comment:
"..allows one/you/the user to pass in ..." or "allows Modifiers to be passed in"

Here's a limerick for this occasion:
```
Though I shouldn't, I'll object!
There's a nit that must be pecked!
  You may shrug,
  But it's a bug:
The comment parser breaks without a subject!
```